ios/RN/RNCamera.m

```diff
@@ -94,6 +95,36 @@ - (id)initWithBridge:(RCTBridge *)bridge
     }
     return self;
 }
+-(float) getMaxZoomFactor:(AVCaptureDevice*)device {
+    float maxZoom;
+    if(self.maxZoom > 1){
+        maxZoom = MIN(self.maxZoom, device.activeFormat.videoMaxZoomFactor);
+    }else{
+        maxZoom = device.activeFormat.videoMaxZoomFactor;
+    }
+    return maxZoom;
+}
+
+-(void) handlePinchToZoomRecognizer:(UIPinchGestureRecognizer*)pinchRecognizer {
+    const CGFloat pinchVelocityDividerFactor = 5.0f;
+
+    if (pinchRecognizer.state == UIGestureRecognizerStateChanged) {
+        AVCaptureDevice *device = [self.videoCaptureDeviceInput device];
+        if(device == nil){
+            return;
+        }
+        NSError *error = nil;
+        float maxZoom = [self getMaxZoomFactor:device]
```

Comment:
Hi! Great work but a semicolon is missing, it won't build at this moment.

Reply:
try the latest version